embed/embed.go

```diff
@@ -374,7 +375,10 @@ func {{.FunctionPrefix}}FSByte(useLocal bool, name string) ([]byte, error) {
 			return nil, err
 		}
 		b, err := ioutil.ReadAll(f)
-		f.Close()
```

Comment:
I think it's more important to return the ReadAll error, so I'd prefer `_ = f.Close()`, which should satisfy the linter.